doc/ci-cd/deployment.md

```diff
@@ -126,7 +126,8 @@ The CI will detect the tag `patch/vX.X.X` and launch the lanes `deploy-ios-stagi
 - Figure out which version (and then tag) you want to deploy. If you patched a version Staging, it could be `patch/v1.X.Y`
 - `yarn trigger:production:deploy <tag>`
 
-This will create a tag `prod-hard-deploy`. The CI will detect the tag and launch the lane [`hard-deploy-android-production`](../../.github/workflows/dev_on_push_workflow_main.yml#L173) & [`hard-deploy-ios-production`](../../.github/workflows/dev_on_push_workflow_main.yml#L183)
+This will create a tag `prod-hard-deploy`. The CI will detect the tag and launch the lane to deploy the production web version.
```

Comment:
de ce que j'en ai compris, ce n'est pas Fastlane qui déploie le web

je propose

```suggestion
This will create a tag `prod-hard-deploy`. The CI will detect the tag and launch the job to deploy the production web version.
```

job ou workflow ou autre mais pas lane